demisto_sdk/commands/common/hook_validations/content_entity_validator.py

```diff
@@ -793,16 +793,25 @@ def is_line_ends_with_dot(self, dict_to_test: dict, arg_field: str):
         line_with_missing_dot = ""
         for arg in dict_to_test.get(arg_field, []):
             stripped_description = strip_description(arg.get("description", ""))
-            if (
-                stripped_description
-                and not stripped_description.endswith(".")
-                and not is_string_ends_with_url(stripped_description)
-            ):
+            if self.is_invalid_description_sentence(stripped_description):
                 line_with_missing_dot += f"The argument {arg.get('name')} description should end with a period.\n"
         for output in dict_to_test.get("outputs") or []:
             stripped_description = strip_description(output.get("description", ""))
-            if not stripped_description.endswith(".") and not is_string_ends_with_url(
-                stripped_description
-            ):
+            if self.is_invalid_description_sentence(stripped_description):
                 line_with_missing_dot += f"The context path {output.get('contextPath')} description should end with a period.\n"
         return line_with_missing_dot
+
+    def is_invalid_description_sentence(self, stripped_description: str) -> bool:
+        """
+        Args:
+            stripped_description: (str) a description or comment section from script / integration yml.
+        Return False if none of the following conditions are met:
+        - The description string doesn't exist/empty.
+        - The description string ends with a dot.
+        - The description string ends with an URL.
+        """
+        return not (
+            not stripped_description
+            or stripped_description.endswith(".")
+            or is_string_ends_with_url(stripped_description)
+        )
```

Comment:
@GuyAfik

Reply:
I think the previous implementation is better, more readable IMO